### PR TITLE
perf(documentation): Adding missing content to guide.md

### PR DESCRIPTION
**Pull request by YurelyCamacho**
## Pull Request description
Add the missing information to complete the user guide on the elements in the TUI.

- Improvements to the current text

The added information corresponds to:

- Introduction for The author of project and The information about the project sections.
- Command Line Interfaces: click and argparse.
- Documentation engine: mkdocs, sphinx and jupyter-book.
- Virtual environment: conda.
- Code formatter: blue.
- Code style and logic (code quality): isort, McCabe, Vulture, pydocstyle.
- Pre-commit verification: pre-commit.

## Pull Request checklists

This PR is a:
- [ ] bug-fix
- [ ] new feature
- [x] maintenance

About this PR:
- [ ] it includes tests.
- [ ] the tests are executed on CI.
- [ ] the tests generate log file(s) (path).
- [ ] pre-commit hooks were executed locally.
- [ ] this PR requires a project documentation update.

Author's checklist:
- [ ] I have reviewed the changes and it contains no misspelling.
- [ ] The code is well commented, especially in the parts that contain more complexity.
- [ ] New and old tests passed locally.

## Reviewer's checklist

Copy and paste this template for your review's note:

```
## Reviewer's Checklist

- [ ] I managed to reproduce the problem locally from the `main` branch
- [ ] I managed to test the new changes locally
- [ ] I confirm that the issues mentioned were fixed/resolved .
```

### CODE REVIEW

File: docs/guide.md [hunk 1]
```diff
@@ -500,6 +743,10 @@ Overall, Docker, Podman, and Kubernetes are powerful tools for managing and
 deploying containerized applications, and can provide a streamlined and
 efficient platform for running Python projects.
 
+In case you do not want to include alguna de estas opciones in your
```
Comment (Anavelyz) [inline docs/guide.md:746]:
```suggestion
In case you do not want to include DevOps in your
```

File: docs/guide.md [hunk 2]
```diff
@@ -71,20 +71,104 @@ arrow keys to navigate through the list. If you accidentally select an option
...
 
 The following is a description of the options in the TUI.
+
 ## The author of project
 
-- **author_name**
-- **author_email**
+With SciCookie, you can easily add information about the author of the
+project by filling in the **author_full_name** and **author_email** fields,
+which refer to the name and email address of the person or organisation
```
Comment (Anavelyz) [inline docs/guide.md:79]: @xmnlab Are we going to use British English as the standard or American English? 🤔

**Review by Anavelyz — COMMENTED** [review]
Thank you @YurelyCamacho, I left some comments…

File: docs/guide.md (hunk 2 above)
Comment (YurelyCamacho) [reply docs/guide.md:79]: Thanks @Anavelyz

Comment (xmnlab) [reply docs/guide.md:79]: I personally prefer the American English, do you have any preference?

**Review by xmnlab — APPROVED** [review]
from my side, it looks good. thanks @YurelyCamacho 
just waiting the discussion about the English vs American standard

**Comment by YurelyCamacho** [comment]
@xmnlab Well, I think the American is better. I'm going to fix some words

**Comment by xmnlab** [comment]
LGTM! thanks for working on that @YurelyCamacho !

**Pull request merged**